WordPress-VIP-Go/ruleset.xml

```diff
@@ -50,7 +50,6 @@
 	<rule ref="WordPressVIPMinimum.VIP.FetchingRemoteData.fileGetContentsUknown">
 		<type>error</type>
 		<Severity>5</Severity>
-		<message>%s() is uncached. If this is being used to query a remote file please use wpcom_vip_file_get_contents() instead. If it's used for a local file please use WP_Filesystem instead. Read more here: https://vip.wordpress.com/documentation/using-wp_filesystem-instead-of-direct-file-access-functions/</message>
```

Comment:
This contains a documentation link that isn't present in the message in the sniff. Is it still of value to keep the link, either in the sniff (for WPCOM + Go), or this ruleset (just Go)?

Reply:
I don't think it's of value to keep the link since it's marked as "VIP Go Platform Specific".  If anything, I'm thinking that we should use https://codex.wordpress.org/Filesystem_API.